webapp-django/crashstats/signature/static/signature/js/signature_tab_correlations.js

```diff
@@ -0,0 +1,187 @@
+/*global SignatureReport: true, Accordion: true, socSortCorrelation:true */
```

Comment:
This file is mostly a copy of https://github.com/mozilla/socorro/blob/master/webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report_list_correlations.js with some adjustments to make it work in the context of Signature Report.